### ISSUE TITLE
Question/suggestion - all versions

### ISSUE THREAD

**Issue by MoonCutter2B** [Lunatrius/Schematica#418]
While having a schematic loaded and printing I placed a sign manually. The sign was placed without me having a chance to write anything on it. And that is probably logical since printing would otherwise be interrupted every time a sign is placed. 
And I could not right click on it after it was placed either, a feature that is used by some other mods.
So is it possible that the right click feature on stuff is not restored properly after a schematic is unloaded?
Restarting minecraft restores this feature until I load a new schematic.

**Comment by Pokechu22** [comment]
The way you've written this confuses me.  Are you asking for adding support to edit the sign text after it's placed automatically by the printer, or are you reporting a bug where after using the printer, normal sign placement doesn't let you edit it?

**Comment by MoonCutter2B** [comment]
Yes,  I tried to report a possible bug, but it is not obvious if it is Schematica or the other mods that have a problem. 
All I know is that everything works perfectly until I load a schematic and while that is loaded right clicking on stuff, specially signs and chests that have added functionality stops working even if I'm far away from where the schematic is loaded.
Once I unload the schematic normal signs start working again but the signs etc from these other mods never recover until I restart the game.
